packages/kbn-pm/src/cli.ts

```diff
@@ -11,6 +11,7 @@ function help() {
     .map(commandName => commands[commandName])
     .map(command => `${command.name} - ${command.description}`);
 
+  /* tslint:disable-next-line no-console */
   console.log(dedent`
```

Comment:
__question:__ can't we just create `log.ts` that exposes function with with `console.log` signature and add this annotation only there or just keep `tslint.yaml` with `no-console: false`? Just don't want to embrace these `/* tslint: something */`.

Reply:
❤️

Reply:
Yeah, I hope most people are averse to the `tslint:disable` directives and will avoid `console.log()` usage because of it, but I agree that having a suitable replacement is better than spreading the directives all over `kbn-pm`